lib/xboxkrnl/xboxkrnl.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include <xboxkrnl/xboxdef.h>
+#include <xboxkrnl/xboxnt.h>
```

Comment:
Reminder: These will probably need this in the future:

```
/* stop clang from crying */
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wlanguage-extension-token"

/* MSVC-compatibility for structure-packing */
#pragma ms_struct on
```

*(Can be done after this PR, but it's something worth noting before we add even more)*